src/release/issue-triaging.md

```diff
@@ -0,0 +1,80 @@
+
+Tracking issues (label `C-tracking-issue`) don't fit into this procedure and are treated differently.
+
+## Motivation
+
+The `rust-lang/rust` repository has thousands of issues and hundreds of people working on it.
+It is impossible for all people to check and solve issues. The goals of triaging are connecting
+issues to the relevant people, and helping them be more effective at fixing the issue.
+
+In practice, it is unrealistic for all issues to be solved quickly and found the by right people.
+Through applications of labels we make the issue tracker more searchable for future reference,
+so that people in the future have an easier time finding related issues or issues they are interested
+in working on.
+
+Triaging can be done by **everyone**, no matter your permissions. We encourage everyone to help here,
+as triaging work is highly parallelizable and easy to get started with.
+
+## Initial triaging
+
+When an issue is opened, it gets the `needs-triage` label. This ensures that every issue gets an initial
+look and that no issue is forgotten, or that when it is forgotten, it is at least visibly forgotten by still having the label.
+
+`needs-triage` is an initial checkpoint. The effort needed to get an issue past the label should be minimal.
+
+To do the initial triage and remove the `needs-triage` label, the following conditions should be fulfilled/considered.
+It's okay if not all of these are always considered, treat it as guideline, not a hard check list. It is also not exhaustive.
+
+- The issue should make sense, that is it should present a problem.
+    - For example, if an issue is a question about Rust in general, the issue should be closed and the user redicted to URLO/Discord.
+      You can of course answer the question too :) (but make sure to mention that the user should go to URLO/Discord next time).
+- Add appropriate labels ([Labels](#labels))
+    - Specifically, `T-*` and `C-*` are the most relevant
+- If the issue contains no reproduction but needs one, ask for one and add the `S-needs-repro` label
+- If the issue could benefit from bisecting the regression, add `E-needs-bisection` (or do the bisection yourself)
+- Does this issue require nightly? Add `requires-nightly`.
+- Is the issue a regression? Apply the `regression-untriaged` label (or figure out what regression it is exactly)
+- If you happen to know people who this issue is relevant to, ping them.
+    - For example, write `cc @ThatPerson` if `ThatPerson` has been working a lot on the problematic feature recently
+- Does this issue require incomplete or internal features? Add `requires-{incomplete,internal}-features`.
+
+For applying and removing labels, unprivileged users can use rustbot.
+For example, `@rustbot label +T-compiler +C-bug +A-linkage +O-macos -needs-triage`.
+
+To see a list of all labels, check out the "labels" page next to the search bar in the issue tracker.
+
+## Further triaging
```

Comment:
(Minor) Could you add a sentence encouraging people to <kbd>needs-triage</kbd> when closing an issue or when the issue has already been closed (or re-open the latter if deemed necessary) since closing an issue is a form of triage?

Although I've noticed that some others do this as well from time to time, I find myself mass-removing <kbd>needs-triage</kbd> from closed issues every so often.

Reply:
Could you elaborate a bit on why this is necessary? What is the use case for including closed issues in needs-triage queries?

Reply:
It's not super important.
(1) Sometimes (on the rarer side) issue authors suddenly close their issue by themselves, e.g., if they find a workaround or if to them it appears the issue has been fixed. In such case, a closed <kbd>needs-triage</kbd> issue might indicate that rustc devs should take another look at it.
(2) More often, <kbd>needs-triage</kbd> issues get fixed by a PR (via *Fixes*) but now the closed issue doesn't have any labels that might be helpful to future people searching the issue tracker for open and/or closed issues which they might do to get the full picture of an area of the compiler. E.g., in my early rustc dev days I used to look for issues similar to one I wanted to fix and see how they were fixed.

Reply:
I'm not sure whether this is worth it. The best way to do this is either automation (rustbot) or just mass-removing the label on closed issues by hand (which cannot be done by unprivileged users). I don't think anyone should be running around leaving rustbot comments on closed issues to remove needs-triage. I'll just keep this out for now.